pkgs/sketch_pad/lib/keys.dart

```diff
@@ -11,13 +11,19 @@ import 'theme.dart';
 bool get _mac => defaultTargetPlatform == TargetPlatform.macOS;
 bool get _nonMac => defaultTargetPlatform != TargetPlatform.macOS;
 
-// key activators
+// ## Key activators
 
-final ShortcutActivator reloadKeyActivator = SingleActivator(
-  LogicalKeyboardKey.keyS,
+final ShortcutActivator runKeyActivator1 = SingleActivator(
```

Comment:
We might want to make sure we're following best-practices for shortcuts and actions: https://docs.flutter.dev/ui/interactivity/actions-and-shortcuts

Reply:
👍 - I filed https://github.com/dart-lang/dart-pad/issues/2913 to track this